Mathlib/Data/ENat/Basic.lean

```diff
@@ -405,60 +405,50 @@ protected theorem map_add {β F} [Add β] [FunLike F ℕ β] [AddHomClass F ℕ
     (f : F) (a b : ℕ∞) : (a + b).map f = a.map f + b.map f :=
   WithTop.map_add f a b
 
+@[to_additive]
```

Comment:
I am not sure to understand why we want to add this lemma that seems only a special case of `WithTop.map_eq_one_iff` (and likewise for the one below).

Reply:
Following [this poll](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/Defeq.20abuse.20with.20ENat/near/482110206), all `WithTop.map` functions have to be copied over to `ENat.map` if you want to use them